### ISSUE TITLE
Searching for a movie returns the movie itself as the first recommendation. Is that intentional ? Also, the recommendations are in a dictionary (JSON) format, a different representation with a UI would be nice.

### ISSUE THREAD

**Issue by Ojjie** [Sahith02/movie-suggest-api#2]

**Comment by Sahith02** [comment]
Hey, Thanks for your insight on the first movie recommended being the same movie you searched for. According to the model predictions, this has been handled. Would you mind telling me which movie it is that is giving the same recommendation as you searched for? I will look into it.

And for your second query, as of right now, the API returns only a JSON format of the recommendations. I am currently working on making UI for that page as well.

**Comment by Ojjie** [comment]
No problem! So, I searched for a few popular movies and the recommendations were different from the movie itself and were pretty good. But upon searching for some movies like Othello, Wild Card (2015) , Never Back Down (2008) etc. (taking inspiration from the background wallpaper xD), the search returned the same movie as the first recommendation. I have attached a screenshot of the issue as well, please do look into it since it seems like the model is doing fine for most of the movies, but is printing the same movie as a recommendation for a few.

![Screenshot 2020-06-19 at 1 14 09 PM](https://user-images.githubusercontent.com/40062564/85110083-fad9b480-b22f-11ea-9b18-630ce57cc566.png)
![Screenshot 2020-06-19 at 1 14 16 PM](https://user-images.githubusercontent.com/40062564/85110103-02995900-b230-11ea-936e-f075ff1c7dfe.png)

**Comment by Sahith02** [comment]
Okay, I've seen the recommendations for these movies and it seems to be showing the searched movie only for a few movies. I will correct that very soon. Thanks for raising this request.

**Comment by Sahith02** [comment]
I've refactored the entire code and now it doesn't show the same movies again as before. One problem is the dataset used has a mix of Hindi and English movies so it will not only recommend Hollywood films but a few Bollywood as well (trying to filter it out for now for universal use).  I've also added more details in the JSON response object such as the movie poster link, IMDB id, etc. Currently working on the UI for the response page and will add that in soon. So I will be closing this issue for now.
